dstyle.dd

```diff
@@ -135,7 +135,22 @@ ubyte asciiChar;
 -------------------------------
     )
 
+    $(DT $(LNAME2 naming_udas, User Defined Attributes))
```

Comment:
User-defined, see https://github.com/dlang/dlang.org/pull/2003#discussion_r157333463

Reply:
It's called "User Defined" everywhere in the spec https://dlang.org/spec/attribute.html#uda.

Reply:
Updated to `User-Defined` since it's in a title.

Reply:
Yeah, we should probably update the spec -> https://github.com/dlang/dlang.org/pull/2077